inc/frontend/namespace.php

```diff
@@ -58,8 +58,24 @@ function output_script() {
  */
 function mangle_images( $content ) {
 	// Find images
-	$searcher = '#<img[^>]+(?:class=[\'"]([^\'"]*wp-image-(\d+)[^\'"]*)|data-gaussholder-id="(\d+)")[^>]+>#x';
-	if ( ! preg_match_all( $searcher, $content, $images, PREG_SET_ORDER ) ) {
+	$searcher = '#<img[^>]+(?:class=[\'"]([^\'"]*wp-image-(\d+)[^\'"]*)|data-gaussholder-id="(\d+)")[^>]+>#x';$preg_match_result = preg_match_all( $searcher, $content, $images, PREG_SET_ORDER );
```

Comment:
```suggestion
	$searcher = '#<img[^>]+(?:class=[\'"]([^\'"]*wp-image-(\d+)[^\'"]*)|data-gaussholder-id="(\d+)")[^>]+>#x';
	$preg_match_result = preg_match_all( $searcher, $content, $images, PREG_SET_ORDER );
```
Not sure why these ended up on one line!